common/utilities/integration_adaptors_logger.py

```diff
@@ -102,6 +102,15 @@ def _format_values_in_map(self, dict_values: dict) -> dict:
         Replaces the values in the map with key=value so that the key in a string can be replaced with the correct
         log format, also surrounds the value with quotes if it contains spaces and removes spaces from the key
         """
+
+        for key in dict_values.keys():
+            if str(key).lower() == "exception":
```

Comment:
I feel like in theory we shouldn't ever really be using something that isn't a `string` for the key anyway, but this check is fine as it is.